src/Engine/Util/Settings.cpp

```diff
@@ -4,49 +4,49 @@ using namespace std;
 
 bool Settings::GetBool(const string& settingName) {
     BoolSetting setting = boolSettings[settingName];
-    
+
     return ini.GetBoolValue(setting.section.c_str(), setting.key.c_str(), setting.defaultValue);
 }
 
 void Settings::SetBool(const string& settingName, bool value) {
     BoolSetting setting = boolSettings[settingName];
-    
+
     ini.SetBoolValue(setting.section.c_str(), setting.key.c_str(), value);
 }
 
 long Settings::GetLong(const string& settingName) {
     LongSetting setting = longSettings[settingName];
-    
+
     return ini.GetLongValue(setting.section.c_str(), setting.key.c_str(), setting.defaultValue);
 }
 
 void Settings::SetLong(const string& settingName, long value) {
     LongSetting setting = longSettings[settingName];
-    
+
     ini.SetLongValue(setting.section.c_str(), setting.key.c_str(), value);
 }
 
 double Settings::GetDouble(const string& settingName) {
     DoubleSetting setting = doubleSettings[settingName];
-    
+
     return ini.GetDoubleValue(setting.section.c_str(), setting.key.c_str(), setting.defaultValue);
 }
 
 void Settings::SetDouble(const string& settingName, double value) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'SetDouble' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)